typings/balena-sdk/models.d.ts

```diff
@@ -258,7 +258,7 @@ export interface Device {
 	download_progress?: number;
 	ip_address: string | null;
 	mac_address: string | null;
-	is_accessible_by_support_until__date: string;
+	is_accessible_by_support_until__date: string | number | null;
```

Comment:
This is changed because the sdk send both `number` and `null` at different points - you can check where by reverting this change and running tsc

Reply:
I changed how this is done in the end